packages/wp-now/src/tests/execute-php-file.spec.ts

```diff
@@ -0,0 +1,38 @@
+import fs from 'fs-extra';
+import path from 'path';
+import { executePHPFile } from '../execute-php-file';
+import { SupportedPHPVersions } from '@php-wasm/universal';
+
+const exampleDir = path.join(__dirname, 'execute-php-file');
+
+test('php file execution in index mode', async () => {
+	const resultFilePath = path.join(exampleDir, 'hello-world-result.txt');
+	// reset result file
+	fs.writeFileSync(resultFilePath, '');
+
+	const result = await executePHPFile(
+		path.join(exampleDir, 'hello-world.php')
+	);
+	expect(result.name).toBe('ExitStatus');
+	expect(result.status).toBe(0);
+	const output = fs.readFileSync(resultFilePath, 'utf8');
+	expect(output).toBe('Hello World!');
+});
+
+test('php file execution for each PHP Version', async () => {
+	const resultFilePath = path.join(exampleDir, 'php-version-result.txt');
+
+	for (const phpVersion of SupportedPHPVersions) {
```

Comment:
Can we add explicit tests here? Assertions inside of `for` always give me concern because they can accidentally stop working.

Reply:
I removed the for loop here:
https://github.com/WordPress/wordpress-playground/pull/336/commits/fe410c82affc7c368a28e1dece54c5f0929862fa

Thanks for the advice.